client/js/components/user/DpUserFormFields.vue

```diff
@@ -71,25 +71,25 @@
       </label>
       <dp-multiselect
         v-if="hasPermission('area_organisations')"
-        :placeholder="Translator.trans('search.three.signs')"
-        :value="currentUserOrga"
-        :searchable="true"
+        :id="userId + ':organisationId'"
+        :close-on-select="true"
+        data-cy="organisation"
         label="name"
         :loading="isLoading"
-        :close-on-select="true"
-        @select="changeUserOrga"
-        required
         :options="initialOrgaSuggestions"
-        :id="userId + ':organisationId'"
+        :placeholder="Translator.trans('search.three.signs')"
+        ref="orgasDropdown"
+        required
+        :searchable="true"
```

Comment:
Isn't `true` the default value for `searchable`?

Reply:
Yes, it is, thanks. Done in [da036c6](https://github.com/demos-europe/demosplan-core/pull/1393/commits/da036c6b771df8c60534af5fd97fbd871855915f)